[RU] Update `Medals` (#8000)
ppy/osu-wiki: wiki/Medals/ru.md

# Медали

> Медали, ранее известные как *достижения*, — значки в профиле игрока, показывающие, что пользователь достиг необычной цели. В данный момент, существует 280 медалей для получения, разделённых на 8 групп: [Beatmap Packs](#beatmap-packs), [Skill](#skill), [Dedication](#dedication), [Mod Introduction](#mod-introduction), [Hush-Hush](#hush-hush), [Beatmap Spotlights](#beatmap-spotlights), [Seasonal Spotlights](#seasonal-spotlights), and [Beatmap Challenge Packs](#beatmap-challenge-packs).

Reviewer: ```suggestion
**Медали**, ранее известные как *достижения*, — значки в профиле игрока, показывающие, что пользователь достиг необычной цели. В данный момент, существует **280** медалей для получения, разделённых на 8 групп: [Beatmap Packs](#beatmap-packs), [Skill](#skill), [Dedication](#dedication), [Mod Introduction](#mod-introduction), [Hush-Hush](#hush-hush), [Beatmap Spotlights](#beatmap-spotlights), [Seasonal Spotlights](#seasonal-spotlights), и [Beatmap Challenge Packs](#beatmap-challenge-packs).
```

## Beatmap Packs

> Медали группы **Beatmap Packs** могут быть получены за **прохождение до конца хотя бы одной из сложностей на каждой карте** из [Тематических Сборок](https://osu.ppy.sh/beatmaps/packs?type=theme) или [Сборок Исполнителей/Альбомов](https://osu.ppy.sh/beatmaps/packs?type=artist). 

Reviewer: ```suggestion
Медали группы **Beatmap Packs** могут быть получены за **прохождение до конца хотя бы одной из сложностей на каждой карте** из [Тематических Сборок](https://osu.ppy.sh/beatmaps/packs?type=theme) или [Сборок Исполнителей/Альбомов](https://osu.ppy.sh/beatmaps/packs?type=artist).
```

Reviewer: Don't seems to be exactly same as english (`These medals can be obtained by clearing every beatmap all the way through at least once in any difficulty within the corresponding beatmap pack.` removed ?). You have to stay as much as you can close to the en.md

> Медали группы **Beatmap Packs** могут быть получены за **прохождение до конца хотя бы одной из сложностей на каждой карте** из [Тематических Сборок](https://osu.ppy.sh/beatmaps/packs?type=theme) или [Сборок Исполнителей/Альбомов](https://osu.ppy.sh/beatmaps/packs?type=artist). 

Author: @TicClick Артист это всё же немножко другое, поэтому я бы всё таки использовал слово "исполнитель"!

```suggestion
Медали группы **Beatmap Packs** могут быть получены за **прохождение до конца хотя бы одной сложности на каждой карте** в подборках по [определённой тематике](https://osu.ppy.sh/beatmaps/packs?type=theme) или конкретному [исполнителю или альбому](https://osu.ppy.sh/beatmaps/packs?type=artist).
```

> Медали группы **Beatmap Packs** могут быть получены за **прохождение до конца хотя бы одной из сложностей на каждой карте** из [Тематических Сборок](https://osu.ppy.sh/beatmaps/packs?type=theme) или [Сборок Исполнителей/Альбомов](https://osu.ppy.sh/beatmaps/packs?type=artist). 

Reviewer 2: да, ты всё правильно говоришь, я задумался просто. исполнитель, конечно.

> Все карты должны быть пройдены в режиме игры [osu!](/wiki/Game_mode/osu!) для получения значка, если не указано обратное. Использование [упрощающих модов](/wiki/Game_modifier#difficulty-reduction) или [усложняющих модов](/wiki/Game_modifier#difficulty-increase), как и использование [Spun Out](/wiki/Game_modifier/Spun_Out) разрешено.

Reviewer: ```suggestion
Все карты должны быть пройдены в режиме игры [osu!](/wiki/Game_mode/osu!) для получения значка, если не указано обратное. Использование [понижение сложности](/wiki/Game_modifier#difficulty-reduction) или [повышение сложности](/wiki/Game_modifier#difficulty-increase), как и использование [Spun Out](/wiki/Game_modifier/Spun_Out) разрешено.
```

-> refering to `https://osu.ppy.sh/wiki/ru/Gameplay/Game_modifier`

Однако стоит отметить что содержимое всех Video Game, Anime, Internet! и Rhythm Game Паков сильно изменилось с [их первого появления в 2008](https://osu.ppy.sh/community/forums/topics/1853). Для списка этих паков в их оригинальном состоянии см. [Legacy Beatmap Packs](/wiki/Medals/Legacy_beatmap_packs).

## Skill

> Медали группы **Skill** присуждаются за достижения, основанные на различных навыках, будь это получение определённого [комбо](/wiki/Beatmapping/Combo), достижение определённого ранга в [глобальной таблице лидеров по очкам производительности](https://osu.ppy.sh/rankings/osu/performance), прохождение или [полное комбо](/wiki/Gameplay/Full_combo) на любой [Квалифицированной](/wiki/Beatmap/Category#qualified), [Рейтинговой](/wiki/Beatmap/Category#ranked) или [Одобренной](/wiki/Beatmap/Category#approved) карте в определённом диапазоне [звёздного рейтинга](/wiki/Beatmapping/Star_rating).

Reviewer: ```suggestion
Медали группы **Skill** присуждаются за достижения, основанные на различных навыках, будь это получение определённого [combo](/wiki/Beatmapping/Combo), достижение определённого ранга в [глобальной таблице лидеров по очкам производительности](https://osu.ppy.sh/rankings/osu/performance), прохождение или [full combo](/wiki/Gameplay/Full_combo) на любой [Qualified](/wiki/Beatmap/Category#qualified), [Ranked](/wiki/Beatmap/Category#ranked) или [Approved](/wiki/Beatmap/Category#approved) карте в определённом диапазоне [star rating](/wiki/Beatmapping/Star_rating).
```
-> refering to (https://osu.ppy.sh/wiki/ru/Beatmap/Category), categories don't seems to be translated.
    -> also technical stuff like combo / full combo / star rating don't have to be translated (i guess ? see french translation)

> Медали группы **Skill** присуждаются за достижения, основанные на различных навыках, будь это получение определённого [комбо](/wiki/Beatmapping/Combo), достижение определённого ранга в [глобальной таблице лидеров по очкам производительности](https://osu.ppy.sh/rankings/osu/performance), прохождение или [полное комбо](/wiki/Gameplay/Full_combo) на любой [Квалифицированной](/wiki/Beatmap/Category#qualified), [Рейтинговой](/wiki/Beatmap/Category#ranked) или [Одобренной](/wiki/Beatmap/Category#approved) карте в определённом диапазоне [звёздного рейтинга](/wiki/Beatmapping/Star_rating).

Author: Согласен, "число звёзд" звучит намного приятнее

## Dedication

Медали группы **Dedication** выдаются за достижение определённого количества игр в одном режиме игры.

## Mod Introduction

> *См. также: [Модификатор игры](/wiki/Game_modifier)*

Reviewer: ```suggestion
*См. также: [игровой модификатор](/wiki/Game_modifier)*
```

> *См. также: [Модификатор игры](/wiki/Game_modifier)*

Author: @TicClick Я считаю что лучше использовать одинаковый перевод с тем что находиться на данной странице вики, поэтому предлагаю сделать так:

```suggestion
*См. также: [Игровой модификатор](/wiki/Game_modifier)*
```

> Медали группы **Mod Introduction** могут быть получены за прохождение любой [Квалифицированной](/wiki/Beatmap/Category#qualified), [Рейтинговой](/wiki/Beatmap/Category#ranked) или [Одобренной](/wiki/Beatmap/Category#approved) карты с их соответствующим [модификатором игры (модом)](/wiki/Game_modifier). Эти медали направлены на то, чтобы стимулировать новых игроков попробовать каждый мод и изучить что может предложить игра. Однако, стоит заметить что если игрок будет играть с несколькими модами одновременно, игрок **не** сможет получить *какую-либо* из медалей группы **Mod Introductions** за эту игру.

Reviewer: ```suggestion
Медали группы **Mod Introduction** могут быть получены за прохождение любой [Qualified](/wiki/Beatmap/Category#qualified), [Ranked](/wiki/Beatmap/Category#ranked) или [Approved](/wiki/Beatmap/Category#approved) карты с их соответствующим [игровой модификатор (модом)](/wiki/Game_modifier). Эти медали направлены на то, чтобы стимулировать новых игроков попробовать каждый мод и изучить что может предложить игра. Однако, стоит заметить что если игрок будет играть с несколькими модами одновременно, игрок **не** сможет получить *какую-либо* из медалей группы **Mod Introductions** за эту игру.
```

> Все медали группы **Mod Introduction** могут быть разблокированы в любом [игровом режиме](/wiki/Game_mode) если не указано обратное.

Reviewer: ```suggestion
Все медали группы Mod Introduction могут быть разблокированы в любом [режимы игры](/wiki/Game_mode) если не указано обратное.
```

## Hush-Hush

> Медали данного типа являются "секретными", присуждаясь за выполнение очень специфичных действий, о которых игроку не сообщают напрямую (отсюда и название). Эти шаги могут быть чем угодно, от прохождения карты при особых условиях до получения скора с необычной комбинацией модов. Несмотря на их скрытный характер, есть подсказки, которые помогут всем любопытным охотникам за медалями в их достижении:

Reviewer: ```suggestion
**Hush-Hush mедали** являются "секретными", присуждаясь за выполнение очень специфичных действий, о которых игроку не сообщают напрямую (отсюда и название). Эти шаги могут быть чем угодно, от прохождения карты при особых условиях до получения скора с необычной комбинацией модов. Несмотря на их скрытный характер, есть подсказки, которые помогут всем любопытным охотникам за медалями в их достижении:
```

- Сами названия медалей,
- Краткие описания медалей и

> - "*Флаерные текста*"<!-- https://twitter.com/ephemeralis/status/1068054191602638850 --> медалей (если присутствуют), которые можно посмотреть, наведя на соответствующие карточки.

Reviewer: ```suggestion
- *Флаерные текста*<!-- https://twitter.com/ephemeralis/status/1068054191602638850 --> медалей (если присутствуют), которые можно посмотреть, наведя на соответствующие карточки.
```

Поскольку медали группы **Hush-Hush** должны быть своего рода секретными достижениями, требования для разблокировки не указаны напрямую; Однако, игроки могут [работать вместе чтобы узнать как получить все медали группы **Hush-Hush**](https://osu.ppy.sh/home/news/2016-08-17-new-hush-hush-medals).

## Beatmap Spotlights

Медали группы **Beatmap Spotlights** выдаются за прохождение определённых паков карт выпущенных как часть [Monthly Beatmap Spotlights](https://osu.ppy.sh/home/news/2017-03-18-introducing-to-you-spotlights). Чтобы разблокировать медали в этой категории, игроку нужно поиграть во все карты *в любом из четырёх паков режимов игры* в соответствующем Beatmap Spotlights чарте. Следующие моды разрешены для использования:

> - [Упрощающие моды](/wiki/Game_modifier#difficulty-reduction)

Reviewer: ```suggestion
- [понижение сложности](/wiki/Game_modifier#difficulty-reduction)
```

> - [Усложняющие моды](/wiki/Game_modifier#difficulty-increase)

Reviewer: ```suggestion
- [повышение сложности](/wiki/Game_modifier#difficulty-increase)
```

- [xK](/wiki/Game_modifier/xK), [Mirror](/wiki/Game_modifier/Mirror) и [Spun Out](/wiki/Game_modifier/Spun_Out)

Несмотря на то, что Monthly Beatmap Spotlights были прекращены с апреля 2018 года, будучи заменёнными [Seasonal Spotlights](https://osu.ppy.sh/home/news/2018-11-01-beatmap-spotlights-summer-2018), все медали, относящиеся к ним, всё ещё доступны для получения и по сей день.

## Seasonal Spotlights

> Медали группы **Seasonal Spotlights medals** выдаются за прохождение определённых паков карт выпущенных как часть [Seasonal Spotlights](https://osu.ppy.sh/home/news/2018-11-01-beatmap-spotlights-summer-2018). Чтобы разблокировать медали в этой категории, игроку нужно поиграть во все карты *в любом из четырёх паков режимов игры* в соответствующем Seasonal Spotlights чарте. Следующие моды разрешены для использования:

Reviewer: ```suggestion
Медали группы **Seasonal Spotlights** выдаются за прохождение определённых паков карт выпущенных как часть [Seasonal Spotlights](https://osu.ppy.sh/home/news/2018-11-01-beatmap-spotlights-summer-2018). Чтобы разблокировать медали в этой категории, игроку нужно поиграть во все карты *в любом из четырёх паков режимов игры* в соответствующем Seasonal Spotlights чарте. Следующие моды разрешены для использования:
```

> - [Упрощающие моды](/wiki/Game_modifier#difficulty-reduction)

Reviewer: ```suggestion
- [понижение сложности](/wiki/Game_modifier#difficulty-reduction)
```

> - [Усложняющие моды](/wiki/Game_modifier#difficulty-increase)

Reviewer: ```suggestion
- [повышение сложности](/wiki/Game_modifier#difficulty-increase)
```

- [xK](/wiki/Game_modifier/xK), [Mirror](/wiki/Game_modifier/Mirror) и [Spun Out](/wiki/Game_modifier/Spun_Out)

Несмотря на то, что Seasonal Spotlights были прекращены с начала 2020 года, будучи замещёнными [теперешней системой Beatmap Spotlights](/wiki/Beatmap_Spotlights), все медали, относящиеся к ним, всё ещё доступны для получения и по сей день.

## Beatmap Challenge Packs

> Медали группы **Beatmap Challenge Packs**, или альтернативно группа [*Skill Packs*](https://osu.ppy.sh/home/news/2020-11-20-featured-artist-beatmap-updates-from-the-mappers-guild#skill-packs) выдаются за прохождение самых высоких сложностей существующих во всех картах представленных в определённых [Тематических Сборках](https://osu.ppy.sh/beatmaps/packs?type=theme) или [Сборках Исполнителей/Альбомов](https://osu.ppy.sh/beatmaps/packs?type=artist) *без использования каких либо [упрощений сложности](/wiki/Game_modifier#difficulty-reduction) или [специальных](/wiki/Game_modifier#special) модов*. Эти сборки карт специально подобраны для теста понимания каждым игроком различных игровых аспектов на определённом уровне навыков, например в рамках определённого [звёздного рейтинга](/wiki/Beatmapping/Star_rating) или предела сложности.

Reviewer: ```suggestion
Медали группы **Beatmap Challenge Packs**, или альтернативно группа [*Skill Packs*](https://osu.ppy.sh/home/news/2020-11-20-featured-artist-beatmap-updates-from-the-mappers-guild#skill-packs) выдаются за прохождение самых высоких сложностей существующих во всех картах представленных в определённых [Theme Beatmap Packs](https://osu.ppy.sh/beatmaps/packs?type=theme) или Artist/Album Beatmap Packs](https://osu.ppy.sh/beatmaps/packs?type=artist) *без использования каких либо [понижение сложности](/wiki/Game_modifier#difficulty-reduction) или [oсобые моды](/wiki/Game_modifier#special) модов*. Эти сборки карт специально подобраны для теста понимания каждым игроком различных игровых аспектов на определённом уровне навыков, например в рамках определённого [star rating](/wiki/Beatmapping/Star_rating) или предела сложности.
```

> ---

Reviewer: ```suggestion
```

> no_native_review: true

Reviewer: ```suggestion
```

> ---

Reviewer: ```suggestion
```

> 

Reviewer: ```suggestion
```
